client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverUri.java

```diff
@@ -280,6 +281,20 @@ public void testUriWithSslEnabled()
         assertThat(properties.getProperty(SSL_TRUST_STORE_PASSWORD.toString())).isNull();
     }
 
+    @Test
+    public void testSqlPath()
+            throws SQLException
+    {
+        assertInvalid("jdbc:trino://localhost:8080?path=catalog.schema.whatever", "Connection property 'path' has invalid syntax, should be [catalog].[schema] or [schema]");
+        assertInvalid("jdbc:trino://localhost:8080", properties("path", "catalog.schema.whatever"), "Connection property 'path' has invalid syntax, should be [catalog].[schema] or [schema]");
+
+        assertThat(createDriverUri("jdbc:trino://localhost:8080?path=catalog.schema").getPath()).hasValue(ImmutableList.of("catalog.schema"));
+        assertThat(createDriverUri("jdbc:trino://localhost:8080?path=catalog,catalog2").getPath()).hasValue(ImmutableList.of("catalog", "catalog2"));
```

Comment:
SQL path is a list of schemas, so the names used in these tests are misleading. Change to “schema1”, “schema2”, etc?

Reply:
it's catalog and schema or just schema (at least according to the docs):
```
Define a collection of paths to functions or table functions in specific catalogs and schemas for the current session.

Each path-element uses a period-separated syntax to specify the catalog name and schema location <catalog>.<schema> of the function, or only the schema location <schema> in the current catalog. The current catalog is set with [USE](https://trino.io/docs/current/sql/use.html), or as part of a client tool connection. Catalog and schema must exist.
```

Reply:
Yes, that's what I meant by "schema" -- the leaf elements have to be schemas. You can't have a naked catalog in the path.

Reply:
Makes sense :)